lib/features/login/data/extensions/list_account_cache_extensions.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:collection/collection.dart';
+import 'package:core/utils/app_logger.dart';
+import 'package:tmail_ui_user/features/login/data/extensions/account_cache_extensions.dart';
+import 'package:tmail_ui_user/features/login/data/model/account_cache.dart';
+
+extension ListAccountCacheExtension on List<AccountCache> {
+  List<AccountCache> unselected() => map((account) => account.unselected()).toList();
+
+  List<AccountCache> removeDuplicated() {
+    final listAccountId = map((account) => account.accountId).whereNotNull().toSet();
```

Comment:
What happens for this case: id is different but all other fields are the same
<img src="https://user-images.githubusercontent.com/80730648/277411987-c1e6caa9-3d70-4e2b-8799-f8b34025f567.png"/>

Reply:
It will delete duplicates. Please see unit test